### PR TITLE
Changed Messages layout and fixed Friends list layout

### PR DESCRIPTION
**Pull request by Michal-Szczepaniak**
Messages layout is now working without constant use of scrollToBottom and i changed text field behaviour to send messages on enter, lowered it to remove empty space and removed send button since enter sends messages also made header static on top so you dont have to scroll all the way up to see if friend is active.

### CODE REVIEW

File: qml/components/AltInputItem.qml
Comment (almindor) [inline qml/components/AltInputItem.qml:51]: We can't use this. People need to be able to send multi-line messages as per std. messaging clients. The PR looks good otherwise but we need to keep the "push button to send" style. Can you change this section?

**Review by almindor — CHANGES_REQUESTED** [review]
Multi-line sending is required

**Comment by Michal-Szczepaniak** [comment]
Sure but i got better idea. If not sure which is better then make it a option is ny motto in gui xD So ill make it an option since i prefer fast short messages i think it will be good solution

**Comment by Michal-Szczepaniak** [comment]
@almindor Could you review that? It should be okay now

**Comment by almindor** [comment]
Thanks, will test ASAP and approve if everything seems good.

**Review by almindor — APPROVED** [review]

**Pull request merged**
